### PR TITLE
Update meta.yaml

### PR DESCRIPTION
**Pull request by dgasmith**
Checklist
* [ ] Used a [fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -27,7 +27,7 @@ requirements:
     - pandas
     - requests
     - pyyaml >=5.1
-    - pydantic >=0.32
+    - pydantic >=0.32,<1.0
```
Comment (Lnaden) [inline recipe/meta.yaml]: Don't forget to bump the build number

Comment (dgasmith) [reply recipe/meta.yaml]: Good point!

**Comment by dgasmith** [comment]
Yea, temporary until Fractal 0.12.

**Comment by Lnaden** [comment]
LGTM then!

**Pull request merged**